src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs

```diff
@@ -45,18 +44,18 @@ public IEnumerable<PackageVerifierIssue> Validate(PackageAnalysisContext context
                             if (AssemblyHelpers.IsAssemblyManaged(assemblyPath))
                             {
                                 isManagedCode = true;
-                                var clrStrongName = (IClrStrongName)RuntimeEnvironment.GetRuntimeInterfaceAsObject(
-                                    new Guid("B79B0ACD-F5CD-409b-B5A5-A16244610B92"),
-                                    new Guid("9FD93CCF-3280-4391-B3A9-96E1CDE77C8D"));
-
-                                bool verificationForced;
-                                hresult = clrStrongName.StrongNameSignatureVerificationEx(assemblyPath, true, out verificationForced);
-                                if (hresult == 0)
+                                isStrongNameSigned = true;
+                                var assembly = AssemblyDefinition.ReadAssembly(assemblyPath);
```

Comment:
We need to write the file to disk for `AssemblyLoadContext.GetAssemblyName()` to work. So I'd say it doesn't matter here.